polling_stations/apps/data_collection/management/commands/import_shropshire.py

```diff
@@ -20,16 +21,30 @@ def address_record_to_dict(self, record):
             rec["postcode"] = "TF10 9AP"
             rec["accept_suggestion"] = False
 
-        return rec
+        if uprn == "100071210835":
+            rec["postcode"] = "WV15 6NL"
+            rec["accept_suggestion"] = False
 
-    def station_record_to_dict(self, record):
+        if record.addressline6 in [
+            "SY4 3QR",
+        ]:
+            return None
 
+    def station_record_to_dict(self, record):
         if record.polling_place_id == "23953":
+            # Stanton Lacy Village Hall, Stanton Lacy, Ludlow
             record = record._replace(polling_place_easting="352573")
             record = record._replace(polling_place_northing="280557")
 
         if record.polling_place_id == "23961":
+            # Farlow & Oreton Village Hall, Fox Hill, Oreton, Kidderminster, Worcs
             record = record._replace(polling_place_easting="364411")
             record = record._replace(polling_place_northing="280033")
 
+        if record.polling_place_id == "23723":
+            # Nesscliffe Village Hall
+            record = record._replace(
+                polling_place_postcode="SY4 1AX", polling_place_uprn="10011838276"
+            )
```

Comment:
I don't think any of these are the `polling_place_id`s from the current file. Sadly the councils' internal id's don't persist.

Reply:
Urgh. I thought I updated those :/